src/components/StrategiesListItem.vue

```diff
@@ -18,7 +18,12 @@ defineEmits(['delete', 'edit']);
 <template>
   <BaseBlock slim class="group mb-3 p-4 text-skin-link">
     <div class="items-center justify-between sm:flex">
-      <h3 class="my-0 leading-5" v-text="strategy.name" />
+      <BaseLink
+        hide-external-icon
+        :link="`https://snapshot.org/#/strategy/${strategy.name}`"
```

Comment:
This will not work on demo site

Reply:
Why not?

Reply:
Playground is the same on demo and prod

Reply:
It will open in a new tab

Reply:
But user will think he is on the demo site which add some confusion